### PR TITLE
feat: Add Scroll Spec

### PR DESCRIPTION
**Pull request by datanexus-vincent**
Does this look good? From what I can tell Scroll doesn't have any custom RPC methods on top of the Ethereum EVM spec.

Testing command:
```
./scripts/test_spec_full.sh cookbook/specs/spec_add_scroll.json jsonrpc https://rpc.scroll.io jsonrpc https://sepolia-rpc.scroll.io
```

### CODE REVIEW

**Comment by Yaroms** [comment]
this is looking good.
thanks for contribution.
did the test script work without errors?

File: cookbook/specs/spec_add_scroll.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{
+    "proposal": {
+        "title": "Add Specs: Scroll",
+        "description": "Adding new specification support for relaying Scroll zkEVM data on Lava",
+        "specs": [
+            {
+                "index": "SCRLL",
```
Comment (Yaroms) [inline cookbook/specs/spec_add_scroll.json:7]: "index": "SCROLL"
just one letter and it would make it more clear

**Comment by datanexus-vincent** [comment]
The test script did work without errors, though I did have to manually set the `delegate-limit` flag for the `stake-provider` command in the script. It doesn't seem like the `$STAKE` variable was defined.

File: cookbook/specs/spec_add_scroll.json (hunk 1 above)
Comment (datanexus-vincent) [reply cookbook/specs/spec_add_scroll.json:7]: Sounds good, I'll make this change. I wasn't sure how strict the "remove vowels" rule was that was mentioned in the spec docs.

**Review by Yaroms — APPROVED** [review]

**Pull request merged**